src/Symfony/Component/Mailer/Bridge/Sendgrid/README.md

```diff
@@ -16,6 +16,39 @@ MAILER_DSN=sendgrid+api://KEY@default
 where:
  - `KEY` is your Sendgrid API Key
 
+
+Webhook:
+--------
+Create route:
+```yaml
+framework:
+    webhook:
+        routing:
+            sendgrid:
+                service: mailer.webhook.request_parser.sendgrid
+                secret: '!SENDGRID_VALIDATION_SECRET!'
+```
+Create consumer:
+```php
+#[\Symfony\Component\RemoteEvent\Attribute\AsRemoteEventConsumer(name: 'sendgrid')]
+class SendGridConsumer implements ConsumerInterface
+{
+    public function consume(RemoteEvent|MailerDeliveryEvent $event): void
+    {
+        //your code
+    }
+}
+```
+
+Default sendgrid enforces to validate the signature, to disable the signature validation:
+```yaml
+framework:
+    mailer:
+        webhook:
+            sendgrid:
+                validate_signature: false
```

Comment:
I'm wondering whether we actually need this boolean parameter at all. Couldn't we automatically enable the signature validation when a non-empty secret is configured for the webhook ?

Reply:
good point, will look into it